### ISSUE TITLE
No continuation commands on Xdebug Client

### ISSUE THREAD

**Issue by nguyenhuulocla2006** [martomo/SublimeTextXdebug#202]
Hi,

I run Xdebug Client with Sublime Text 3 on Window 10. And there is without any continuation command as such as: Run, Run To Line, Step Over, Step Into, Step Out, Stop, Detach. 
Please give me solution for this issue.
[See my issue at here](https://photos.app.goo.gl/jt6kgorb3EJxAt9Y8)

Setup environment:
- OS: Window 10
- Sublime Text: 3.2.2 Build 3211 
- Xdebug: 2.7.0
- PHP version: 7.3.0

**Comment by meengit** [comment]
@nguyenhuulocla2006, as I understand, this package is, unfortunately!, dead. I recently switched to Debugger < https://packagecontrol.io/packages/Debugger> which works very good in my case.

**Comment by meengit** [comment]
@nguyenhuulocla2006, it is not that easy to start with Debugger. Here is a sample configuration of one of my projects:

```json
{
settings: {
	"debug.configurations":
		[
			{
				"name": "MY-PROJECT",
				"pathMappings":
				{
					"/THE/SERVER/PATH": "/YOUR/LOCAL/PATH"
				},
				"request": "launch",
				"stopOnEntry": false,
				"type": "php"
			}
		]
	}
}
```

The installation is as normal with Packagecontrol.

The *Getting Started Guide* is here: < https://github.com/daveleroy/sublime_debugger/blob/master/docs/setup.md>. You may also have to consider < https://github.com/felixfbecker/vscode-php-debug> because Debugger loads VSCode packages in the background.

**Comment by mercurykd** [comment]
> @nguyenhuulocla2006, it is maybe not that easy to start with Debugger. Here is a sample configuration of one of my _Sublime Projects_:
> 
> ```json
> {
> "settings":{
> 	"debug.configurations":
> 		[
> 			{
> 				"name": "MY-PROJECT",
> 				"pathMappings":
> 				{
> 					"/THE/SERVER/PATH": "/YOUR/LOCAL/PATH"
> 				},
> 				"request": "launch",
> 				"stopOnEntry": false,
> 				"type": "php"
> 			}
> 		]
> 	}
> }
> ```
> 
> The installation is as normal with Packagecontrol.
> 
> The _Getting Started Guide_ is here: https://github.com/daveleroy/sublime_debugger/blob/master/docs/setup.md. You may also have to consider https://github.com/felixfbecker/vscode-php-debug because Debugger loads VSCode packages in the background.

how use it?

**Comment by nguyenhuulocla2006** [comment]
@meengit Please help me step by step to install & config this package on sublime text 3. I installed Debuger as your suggestion but it doesn't work. [See error](https://photos.app.goo.gl/6VU6131NHc7R8jk38)

Note: I installed xdebug and config on php.ini as bellow:
[Xdebug]
zend_extension="C:\xampp\php\ext\php_xdebug-2.7.0-7.3-vc15.dll"
xdebug.remote_enable = 1
xdebug.remote_host = "127.0.0.1"
xdebug.remote_port = 9000
xdebug.remote_handler = "dbgp"
xdebug.remote_mode = req
xdebug.remote_connect_back = 1

**Comment by martomo** [comment]
@nguyenhuulocla2006 Are you still experiencing the reported issue with Xdebug Client version 1.1.1? Can you please let me know, I did not manage to check it out on a Windows machine yet.

**Comment by juangiordana** [comment]
Hi @martomo,

I had no luck with your repository to display the continuation commands. Using @gepeixoto repository ended up working out of the box. I remember that after reinstalling this package a json sublime dependency was installed. but I can't remember the exact name.

Let me know if i can be of more help. Thanks

```sh
cd ~/.config/sublime-text-3/Packages
git clone https://github.com/gepeixoto/SublimeTextXdebug "Xdebug Client"
subl
# Toggle breakpoint > 
```


Linux Mint Cinnamon 19.1
Sublime Text Build 3211
Homestead v9.1.0 (virtual machine)

PHP 7.3.9-1+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Sep  2 2019 12:54:24) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.9, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.9-1+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.2, Copyright (c) 2002-2019, by Derick Rethans

**Comment by aubreypwd** [comment]
I am also experiencing this. I can get it to stop at a breakpoint, but no way to step over, into, etc.

~[](http://aubrey.pw/d/2019/1574866002.png)

**Comment by martomo** [comment]
@juangiordana @aubreypwd Can you please verify if this issue still persists, I believe this should be resolved with Xdebug Client version 1.1.2.

**Comment by juangiordana** [comment]
@martomo I confirm it is working in my environment, installed via Package Control. Thanks!
